fit-curve.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "fit-curve" {
+  type Point<Arr extends number[]> = Arr;
+  type Vector<Arr extends number[]> = Arr;
+  type Curve<Arr extends number[]> = [
+    Point<Arr>,
+    Point<Arr>,
+    Point<Arr>,
+    Point<Arr>
+  ];
+
+  function fitCurve<Arr extends number[] = [number, number]>(
```

Comment:
Do you mind moving (copying) all the JsDocs for exported function into here?

Reply:
Done.